erpnext/stock/doctype/purchase_receipt_item/purchase_receipt_item.json

```diff
@@ -267,6 +265,7 @@
    "width": "100px"
   },
   {
+   "default": "1.00",
```

Comment:
Why add a default value for this?

Reply:
Because it's a required field, and if there is no item retrieved, i.e. and arbitrary item name is typed in, it won't get a value